calico_containers/tests/unit/calicoctl_test.py

```diff
@@ -23,6 +23,7 @@
 from calico_ctl.pool import validate_arguments as pool_validate_arguments
 from calico_ctl.profile import validate_arguments as profile_validate_arguments
 from calico_ctl.container import validate_arguments as container_validate_arguments
+from calico_ctl.utils import *
```

Comment:
I like to avoid using * imports unless you really need to (e.g you're importing 100 things from this file).  Even then, it is preferred to do something like

```
from calico_ctl import utils
```

This prevents possible clashes between imports.